vendor/github.com/openshift/client-go/operator/applyconfigurations/utils.go

```diff
@@ -0,0 +1,462 @@
+// Code generated by applyconfiguration-gen. DO NOT EDIT.
+
+package applyconfigurations
```

Comment:
I'm not sure about why do we need changes in vendor directory without even updating the go.mod.

Could you please update the PR;
* First commit go.mod changes and why do we need it?. I think, overall we shouldn't update go.mod
* Second commit "go mod vendor". 
* Third commit required changes in cli/admin/nodeimage package.

/hold

Reply:
Thank you. That makes sense now. Please ping me for another round of review when the CI passes
/hold cancel